src/com/facebook/buck/jvm/java/AccumulateClassNamesStep.java

```diff
@@ -152,7 +152,7 @@ public InputStream openStream() throws IOException {
             if (existing != null && !existing.equals(value)) {
               throw new IllegalArgumentException(String.format(
                   "Multiple entries with same key but differing values: %1$s=%s and %1$s=%s",
-                  key, value, existing));
+                  key, value));
```

Comment:
This and the next change don't look right, I think the format should be changed to something like `%1$s=%2$s and %1$s=%3$s` (or the positional arguments can be removed and we can just pass `key, value, key, existing`).